src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -483,7 +456,7 @@ export default function Swap({
                 showMaxButton={false}
                 hideBalance={false}
                 fiatValue={fiatValueOutput ?? undefined}
-                priceImpact={showWrap ? undefined : priceImpact}
+                priceImpact={onWrap ? undefined : priceImpact}
```

Comment:
This was a bug fix I did recently, because `showWrap` can be set even if `onWrap` is not set.
Unless you changed this behaviour, we should keep `showWrap`

Reply:
Fixed in the next PR's